mps-sync-plugin-lib/src/main/kotlin/org/modelix/mps/sync/SyncServiceImpl.kt

```diff
@@ -58,15 +55,10 @@ class SyncServiceImpl : SyncService {
             return it
         }
 
+        logger.info { "Connecting to $serverURL" }
         // TODO: use JWT here
-        val modelClientV2: ModelClientV2 = ModelClientV2.builder().url(serverURL.toString()).build()
-        try {
-            logger.info { "Connecting to $serverURL" }
-            modelClientV2.init()
-        } catch (e: ConnectException) {
-            logger.warn { "Unable to connect: ${e.message} / ${e.cause}" }
-            throw e
-        }
+        val modelClientV2 = ModelClientV2.builder().url(serverURL.toString()).build()
+        modelClientV2.init()
 
         logger.info { "Connection to $serverURL successful" }
         activeClients.add(modelClientV2)
```

Comment:
Why is not `ModelSyncService.connectModelServer` used to create and keep track of the client?